yoyodyne/models/__init__.py

```diff
@@ -32,6 +33,7 @@ def get_model_cls(arch: str) -> BaseEncoderDecoder:
         "pointer_generator_transformer": PointerGeneratorTransformerEncoderDecoder,  # noqa: 501
         "transducer": TransducerEncoderDecoder,
         "transformer": TransformerEncoderDecoder,
+        "hmm_lstm": HardAttentionHmm,
```

Comment:
Can you think of a better flag name that more closely matches the name of the module? What about like, `--arch hard_attention` or `--arch hard_monotonic_lstm`? While I sort of understand the connection to HMMs I don't think people know it under that name.

Reply:
Yeah I'm just trying not to confuse people since we also have `attentive lstm`. Honestly their architecture makes more sense just saying it's an HMM. Couldn't we just clarify in the readme?

Reply:
We could, but I think it would be better to call it what the previous literature calls it instead.

Reply:
alrighty. though as minor note will mention this is what ryan and shinje call it.

Reply:
Gonna go with `hard_attention_lstm` and then revisit if we want to add a transformer head in its place.